src/Sulu/Bundle/AdminBundle/Tests/app/config/config.yml

```diff
@@ -15,7 +15,6 @@ security:
                 check_path: sulu_admin.login_check
                 success_handler: sulu_security.authentication_handler
                 failure_handler: sulu_security.authentication_handler
-                csrf_provider: security.csrf.token_manager
```

Comment:
in sulu-standard we have to change following: https://github.com/sulu/sulu-standard/pull/770/files#diff-3a35616aea3903a1edf9541042da2b51R36

in the tests we havent used it so i have removed it.